docs/spl-docs/inputlookup-command.md

```diff
@@ -0,0 +1,142 @@
+# inputlookup command
+
+## Description
+The `inputlookup` command is used to search the contents of a CSV-based lookup table. This command allows you to retrieve and filter data stored in a lookup table.
+
+## Syntax
+
+Required syntax is in **bold**.
+
+| **inputlookup**\
+[append=\<boolean\>]\
+[start=\<boolean\>]\
+[max=\<boolean\>]\
+**\<filename\>**
+[WHERE \<search-query\>]
+
+### Required Arguments
+
+#### \<filename\>
+**Syntax:** `<string>`\
+**Description:**\
+The name of the lookup file. It must end with either `.csv` or `.csv.gz`.
+
+### Optional Arguments
+
+#### append
+**Syntax:** `append=<boolean>`\
+**Description:**\
+If set to `true`, the data from the lookup file is appended to the current results instead of replacing them.\
+This option must be set to `true` if the `inputlookup` command is not the first command in the search.\
+**Default:** `false`
+
+#### start
+**Syntax:** `start=<int>`\
+**Description:**\
+Specify the 0-based offset for the first event to read. For example, if `start=0`, it begins with the first event. If `start=5`, it begins with the sixth event.\
+**Default:** 0
+
+#### max
+**Syntax:** `max=<int>`\
+**Description:**\
+Indicate the maximum number of events to read from the file.\
+**Default:** 1000000000
```

Comment:
nit: it's a little hard to read all the zeros
```suggestion
**Default:** 1000000000 (1 billion)
```

Reply:
addressed in the latest commit.